### ISSUE TITLE
Android: Accessibility - back buttons in search and vault > bin lack appropriate accessible name

### ISSUE THREAD

**Issue by patrickhlauke** [bitwarden/mobile#1915]
### Steps To Reproduce

1. Turn on TalkBack
2. Go to "Search"
3. Focus on the "←" back button (A)
4. Go to "My Vault > Bin"
5. Focus on the "←" back button (B)

### Expected Result

TalkBack should announce both of these as "Back, Button".

### Actual Result

For Search (A), it appears that the button's name is "←", but it's a glyph/character that TalkBack doesn't recognise, so it doesn't announce it as anything other than "... , Button".

For the Bin (B), the button completely lacks an accessible name, and is announced as "Unlabelled, Button"

### Screenshots or Videos

Photos (as can't take on-device screenshots, since Bitwarden suppresses this) of Bitwarden on Android, with TalkBack's speech output visible at the bottom:

![Search back button, showing "[broken character icon], Button" TalkBack output](https://user-images.githubusercontent.com/895831/169381939-29e1daa4-0636-4e55-8f01-91c39dec2e6c.jpg)
![Bin back button, showing "Unlabelled, Button" TalkBack output](https://user-images.githubusercontent.com/895831/169382078-8a7265fb-5165-41e8-824e-6f60c95fe3b4.jpg)

### Operating System

Android

### Operating System Version

11

### Device

Motorola moto g(8)

### Build Version

2.18.0 (4572)

### Beta

- [ ] Using a pre-release version of the application.
